app/models/like.rb

```diff
@@ -19,6 +19,10 @@ def relayable_options
 
   after_destroy do
     self.parent.update_likes_counter
+    participation = Participation.where({ 
+      target_id: self.target_id, author_id: self.author_id
+    }).first()
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.<br>Do not use parentheses for method calls with no arguments.